test/transform-mode.test.js

```diff
@@ -0,0 +1,86 @@
+const { test, snapshot } = require("node:test");
+const { transformSync } = require("../dist/index.js");
+const path = require("node:path");
+
+// Set the path for the snapshots directory
+snapshot.setResolveSnapshotPath((testPath) => {
+	return path.join(
+		__dirname,
+		"snapshots",
+		`${path.basename(testPath)}.snapshot`,
+	);
+});
+
+test("should handle TypeScript decorators", (t) => {
```

Comment:
isnt this already handled in the `transform-decoratorna-names` test?

Reply:
Yeah, I think I copied this